prime-router/src/main/kotlin/azure/GetSubmissions.kt

```diff
@@ -0,0 +1,85 @@
+package gov.cdc.prime.router.azure
+
+import com.microsoft.azure.functions.HttpMethod
+import com.microsoft.azure.functions.HttpRequestMessage
+import com.microsoft.azure.functions.HttpResponseMessage
+import com.microsoft.azure.functions.annotation.AuthorizationLevel
+import com.microsoft.azure.functions.annotation.FunctionName
+import com.microsoft.azure.functions.annotation.HttpTrigger
+import gov.cdc.prime.router.tokens.OktaAuthentication
+import org.apache.logging.log4j.kotlin.logger
+import java.time.OffsetDateTime
+
+/**
+ * Submissions API
+ * Returns a list of Actions from `public.action`.
+ */
+
+class GetSubmissions(
+    submissionsFacade: SubmissionsFacade = SubmissionsFacade.common,
+    oktaAuthentication: OktaAuthentication = OktaAuthentication(PrincipalLevel.SYSTEM_ADMIN)
+) {
+    private val facade = submissionsFacade
+    private val oktaAuthentication = oktaAuthentication
+
+    /**
+     * An Azure Function that is triggered at the `/api/submissions/` endpoint
+     *
+     * @param qLimit a user supplied page size limit overwriting the default value.
+     * @param limit a default value for the number of results to display per page.
+     * @return a list of submission history results.
+     */
+    @FunctionName("getSubmissions")
+    fun run(
+        @HttpTrigger(
+            name = "getSubmissions",
+            methods = [HttpMethod.GET],
+            authLevel = AuthorizationLevel.ANONYMOUS,
+            route = "submissions"
+        ) request: HttpRequestMessage<String?>,
+    ): HttpResponseMessage {
+        val qOrder = request.queryParameters.getOrDefault("order", "DESC")
+        val qCursor = request.queryParameters.getOrDefault("cursor", "${OffsetDateTime.now()}")
```

Comment:
make it clear what this will do, something to do with it being a datetime. The `order` and this could even be combined into mutually exclusive `after` and `before` parameters for example.